### DISCUSSION TITLE
Low Background Blur makes it pixelated?

### DISCUSSION THREAD

**Discussion by NotGumballer91** [ppy/osu#16270] [Q&A] [answered]
https://user-images.githubusercontent.com/64581009/147565345-8ff1ff8a-4b3d-46e6-8584-168979e5df57.mp4

Setting a low blur just only downscales the background without any noticable blur.
Is this like a tradeoff from [#6979](https://github.com/ppy/osu/pull/6979)?

**Comment by bdach** [comment, marked as answer]
Yes, that would be correct. Blur turns out to be quite expensive and as such is performed on smaller framebuffers for speed.
